src/components/Thanks/ThanksPageDonationOnly.vue

```diff
@@ -0,0 +1,277 @@
+<template>
+	<div>
+		<generic-promo-banner
+			v-show="monthlyDonationAmount"
+			:promo-banner-content="promoBannerContent"
+		/>
```

Comment:
Why are we using this instead of a persistent toast message? I don't think we want to generalize this interface into a confirmation type like this without working with design to determine when this is used vs a toast message. The toast was an intentional replacement for this type of interface.

Reply:
Also, KvToast accepts a slot so you should be able to set promoBannerContent within that.

Reply:
Thanks for the heads up! we agreed to use the tip msg for this